lib/count_sim_swaps/main.py

```diff
@@ -0,0 +1,78 @@
+import asyncio
+from datetime import datetime, timezone
+import os
+from pngme.api import AsyncClient
+
+
+async def get_count_of_sim_swaps(
+    api_client: AsyncClient,
+    user_uuid: str,
+    utc_timestamp: datetime,
+    semi_window_days: int,
+) -> int:
```

Comment:
Can we make the return type Optional since we can return `None`

Reply:
You are totally right. How is that the linter is not complaining?!